docs/download_files.md

```diff
@@ -23,7 +25,7 @@ The files associated with each library are (example shown for a library with ID
 - A filtered counts file: `SCPCL000000_filtered.rds`,
 - A quality control report: `SCPCL000000_qc.html`,
 
-Every download also includes a single `libraries_metadata.csv` file containing metadata for all libraries included in the download.
+Every download also includes a single `single_cell_metadata.tsv` file containing metadata for all libraries included in the download.
```

Comment:
I feel like this line should be in both the individual sample and project sections. It seems conspicuously absent from the project information. Or perhaps it should be at ~ line 6, since it applies to both.

Reply:
I actually wonder if all of lines 19-26 (starting with "Sample folders (indicated by the `SCPCS` prefix)" up to " A quality control report") should be in that initial section, since those contents apply to both kinds of downloads?